Docs.jsx

```diff
@@ -1,8 +1,20 @@
-export default function Docs(){
-    return (
-        <div>
-          I am an "expensive" component... which really just means that I was
-          code-split 😉
-        </div>
-    )
-}
\ No newline at end of file
+import './stateSignal';
```

Comment:
do you need this line? This is only to be initialised once where our App.jsx loads